.github/workflows/generate-versions.yml

```diff
@@ -0,0 +1,46 @@
+name: Generate RDF Serializations
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - 'ns/*.ttl'
+
+jobs:
+  generate-versions:
+    name: Generate RDF Versions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # Install Apache Jena tools (including riot)
+      - name: Install Jena
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y apache2-utils openjdk-11-jre-headless
+          curl -L https://dlcdn.apache.org/jena/binaries/apache-jena-4.8.0.tar.gz \
+            | tar xz --strip-components=1 -C /home/runner
+          echo 'export PATH="/home/runner/bin:$PATH"' >> $GITHUB_ENV
+
+      # Convert all .ttl files to RDF/XML and JSON-LD using riot
+      - name: Convert RDF to multiple formats
+        run: |
+          for file in ns/*.ttl; do
+            base=$(basename "$file" .ttl)
+            # Convert Turtle -> RDF/XML
+            /home/runner/bin/riot --syntax=turtle --output=rdfxml ns/${base}.ttl > ns/${base}.rdf
+            # Convert Turtle -> JSON-LD
+            /home/runner/bin/riot --syntax=turtle --output=jsonld ns/${base}.ttl > ns/${base}.jsonld
```

Comment:
It might be nice to use a context such as below to get more useful JSON-LD output:

```json
{
  "id": "@id",
  "type": "@type",
  "dc": "http://purl.org/dc/elements/1.1/",
  "owl": "http://www.w3.org/2002/07/owl#",
  "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
  "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
  "title": {"@id": "dc:title"},
  "description": {"@id": "dc:description"},
  "date": {"@id": "dc:date", "@type": "xsd:date"},
  "comment": {"@id": "rdfs:comment"},
  "domain": {"@id": "rdfs:domain", "@type": "@id"},
  "label": {"@id": "rdfs:label"},
  "range": {"@id": "rdfs:range", "@type": "@id"},
  "isDefinedBy": {"@id": "rdfs:isDefinedBy", "@type": "@vocab"},
  "seeAlso": {"@id": "rdfs:seeAlso", "@type": "@id"},
  "subClassOf": {"@id": "rdfs:subClassOf", "@type": "@id"},
  "subPropertyOf": {"@id": "rdfs:subPropertyOf", "@type": "@id"},
  "imports": {"@id": "owl:imports", "@type": "@id"},
  "versionInfo": {"@id": "owl:versionInfo", "@type": "@id"}
}
```

See the [JSON-LD Playground](https://json-ld.org/playground/#startTab=tab-compacted&json-ld=%7B%22%40graph%22%3A%5B%7B%22%40id%22%3A%22rdf%3Adirection%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22https%3A%2F%2Fwww.w3.org%2FTR%2Fjson-ld11%2F%23the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3ACompoundLiteral%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20base%20direction%20component%20of%20a%20CompoundLiteral.%22%2C%22rdfs%3Alabel%22%3A%22direction%22%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%2C%22dc%3Adescription%22%3A%22This%20is%20the%20RDF%20Schema%20for%20the%20RDF%20vocabulary%20terms%20in%20the%20RDF%20Namespace%2C%20defined%20in%20RDF%201.1%20Concepts.%22%2C%22dc%3Adate%22%3A%222019-12-16%22%2C%22dc%3Atitle%22%3A%22The%20RDF%20Concepts%20Vocabulary%20(RDF)%22%2C%22%40type%22%3A%22owl%3AOntology%22%7D%2C%7B%22%40id%22%3A%22rdf%3ACompoundLiteral%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22https%3A%2F%2Fwww.w3.org%2FTR%2Fjson-ld11%2F%23the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Acomment%22%3A%22A%20class%20representing%20a%20compound%20literal.%22%2C%22rdfs%3Alabel%22%3A%22CompoundLiteral%22%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3AProperty%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20RDF%20properties.%22%2C%22rdfs%3Alabel%22%3A%22Property%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3Alanguage%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22https%3A%2F%2Fwww.w3.org%2FTR%2Fjson-ld11%2F%23the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3ACompoundLiteral%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20language%20component%20of%20a%20CompoundLiteral.%22%2C%22rdfs%3Alabel%22%3A%22language%22%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3AJSON%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22https%3A%2F%2Fwww.w3.org%2FTR%2Fjson-ld11%2F%23the-rdf-json-datatype%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3ALiteral%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20datatype%20of%20RDF%20literals%20storing%20JSON%20content.%22%2C%22rdfs%3Alabel%22%3A%22JSON%22%2C%22%40type%22%3A%22rdfs%3ADatatype%22%7D%2C%7B%22%40id%22%3A%22rdf%3AXMLLiteral%22%2C%22rdfs%3Acomment%22%3A%22The%20datatype%20of%20XML%20literal%20values.%22%2C%22rdfs%3Alabel%22%3A%22XMLLiteral%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3ALiteral%22%7D%2C%22%40type%22%3A%22rdfs%3ADatatype%22%7D%2C%7B%22%40id%22%3A%22rdf%3Arest%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdf%3AList%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3AList%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20rest%20of%20the%20subject%20RDF%20list%20after%20the%20first%20item.%22%2C%22rdfs%3Alabel%22%3A%22rest%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3AList%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20RDF%20Lists.%22%2C%22rdfs%3Alabel%22%3A%22List%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3Afirst%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3AList%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20first%20item%20in%20the%20subject%20RDF%20list.%22%2C%22rdfs%3Alabel%22%3A%22first%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3Anil%22%2C%22rdfs%3Acomment%22%3A%22The%20empty%20list%2C%20with%20no%20items%20in%20it.%20If%20the%20rest%20of%20a%20list%20is%20nil%20then%20the%20list%20has%20no%20more%20items%20in%20it.%22%2C%22rdfs%3Alabel%22%3A%22nil%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AList%22%7D%2C%7B%22%40id%22%3A%22rdf%3Avalue%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Acomment%22%3A%22Idiomatic%20property%20used%20for%20structured%20values.%22%2C%22rdfs%3Alabel%22%3A%22value%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3AAlt%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AContainer%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20containers%20of%20alternatives.%22%2C%22rdfs%3Alabel%22%3A%22Alt%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3ASeq%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AContainer%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20ordered%20containers.%22%2C%22rdfs%3Alabel%22%3A%22Seq%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3ABag%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AContainer%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20unordered%20containers.%22%2C%22rdfs%3Alabel%22%3A%22Bag%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3Aobject%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3AStatement%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20object%20of%20the%20subject%20RDF%20statement.%22%2C%22rdfs%3Alabel%22%3A%22object%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3AStatement%22%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20RDF%20statements.%22%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Alabel%22%3A%22Statement%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3AClass%22%7D%2C%7B%22%40id%22%3A%22rdf%3Apredicate%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3AStatement%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20predicate%20of%20the%20subject%20RDF%20statement.%22%2C%22rdfs%3Alabel%22%3A%22predicate%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3Asubject%22%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdf%3AStatement%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20subject%20of%20the%20subject%20RDF%20statement.%22%2C%22rdfs%3Alabel%22%3A%22subject%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3Atype%22%2C%22rdfs%3Adomain%22%3A%7B%22%40id%22%3A%22rdfs%3AResource%22%7D%2C%22rdfs%3Arange%22%3A%7B%22%40id%22%3A%22rdfs%3AClass%22%7D%2C%22rdfs%3Acomment%22%3A%22The%20subject%20is%20an%20instance%20of%20a%20class.%22%2C%22rdfs%3Alabel%22%3A%22type%22%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdf%3AProperty%22%7D%2C%7B%22%40id%22%3A%22rdf%3APlainLiteral%22%2C%22rdfs%3Acomment%22%3A%22The%20class%20of%20plain%20(i.e.%20untyped)%20literal%20values%2C%20as%20used%20in%20RIF%20and%20OWL%202%22%2C%22rdfs%3Alabel%22%3A%22PlainLiteral%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2FTR%2Frdf-plain-literal%2F%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3ALiteral%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22%40type%22%3A%22rdfs%3ADatatype%22%7D%2C%7B%22%40id%22%3A%22rdf%3AlangString%22%2C%22rdfs%3Acomment%22%3A%22The%20datatype%20of%20language-tagged%20string%20values%22%2C%22rdfs%3Alabel%22%3A%22langString%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2FTR%2Frdf11-concepts%2F%23section-Graph-Literal%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3ALiteral%22%7D%2C%22%40type%22%3A%22rdfs%3ADatatype%22%7D%2C%7B%22%40id%22%3A%22rdf%3AHTML%22%2C%22rdfs%3Acomment%22%3A%22The%20datatype%20of%20RDF%20literals%20storing%20fragments%20of%20HTML%20content%22%2C%22rdfs%3Alabel%22%3A%22HTML%22%2C%22rdfs%3AseeAlso%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2FTR%2Frdf11-concepts%2F%23section-html%22%7D%2C%22rdfs%3AisDefinedBy%22%3A%7B%22%40id%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3ALiteral%22%7D%2C%22%40type%22%3A%22rdfs%3ADatatype%22%7D%5D%2C%22%40context%22%3A%7B%22rdfs%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%22%2C%22rdf%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%2C%22owl%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23%22%2C%22dc%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%22%7D%7D&frame=%7B%22dc%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Fterms%2F%22%2C%22owl%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23%22%2C%22rdf%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%2C%22rdfs%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%22%2C%22dc%3Atitle%22%3A%7B%22%40container%22%3A%22%40language%22%7D%2C%22dc%3Adescription%22%3A%7B%22%40container%22%3A%22%40language%22%7D%2C%22dc%3Adate%22%3A%7B%22%40type%22%3A%22xsd%3Adate%22%7D%2C%22rdfs%3Acomment%22%3A%7B%22%40container%22%3A%22%40language%22%7D%2C%22rdfs%3Adomain%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22rdfs%3Alabel%22%3A%7B%22%40container%22%3A%22%40language%22%7D%2C%22rdfs%3Arange%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22rdfs%3AseeAlso%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22rdfs%3AsubClassOf%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22rdfs%3AsubPropertyOf%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22owl%3Aimports%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22owl%3AversionInfo%22%3A%7B%22%40type%22%3A%22%40id%22%7D%2C%22schema%3AdomainIncludes%22%3A%7B%22%40type%22%3A%22%40vocab%22%2C%22%40container%22%3A%22%40set%22%7D%2C%22schema%3ArangeIncludes%22%3A%7B%22%40type%22%3A%22%40vocab%22%2C%22%40container%22%3A%22%40set%22%7D%2C%22rdfs_classes%22%3A%7B%22%40reverse%22%3A%22rdfs%3AisDefinedBy%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22rdfs_properties%22%3A%7B%22%40reverse%22%3A%22rdfs%3AisDefinedBy%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22rdfs_datatypes%22%3A%7B%22%40reverse%22%3A%22rdfs%3AisDefinedBy%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22rdfs_instances%22%3A%7B%22%40reverse%22%3A%22rdfs%3AisDefinedBy%22%2C%22%40type%22%3A%22%40id%22%7D%7D&context=%7B%22id%22%3A%22%40id%22%2C%22type%22%3A%22%40type%22%2C%22dc%22%3A%22http%3A%2F%2Fpurl.org%2Fdc%2Felements%2F1.1%2F%22%2C%22owl%22%3A%22http%3A%2F%2Fwww.w3.org%2F2002%2F07%2Fowl%23%22%2C%22rdf%22%3A%22http%3A%2F%2Fwww.w3.org%2F1999%2F02%2F22-rdf-syntax-ns%23%22%2C%22rdfs%22%3A%22http%3A%2F%2Fwww.w3.org%2F2000%2F01%2Frdf-schema%23%22%2C%22title%22%3A%7B%22%40id%22%3A%22dc%3Atitle%22%7D%2C%22description%22%3A%7B%22%40id%22%3A%22dc%3Adescription%22%7D%2C%22date%22%3A%7B%22%40id%22%3A%22dc%3Adate%22%2C%22%40type%22%3A%22xsd%3Adate%22%7D%2C%22comment%22%3A%7B%22%40id%22%3A%22rdfs%3Acomment%22%7D%2C%22domain%22%3A%7B%22%40id%22%3A%22rdfs%3Adomain%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22label%22%3A%7B%22%40id%22%3A%22rdfs%3Alabel%22%7D%2C%22range%22%3A%7B%22%40id%22%3A%22rdfs%3Arange%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22isDefinedBy%22%3A%7B%22%40id%22%3A%22rdfs%3AisDefinedBy%22%2C%22%40type%22%3A%22%40vocab%22%7D%2C%22seeAlso%22%3A%7B%22%40id%22%3A%22rdfs%3AseeAlso%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22subClassOf%22%3A%7B%22%40id%22%3A%22rdfs%3AsubClassOf%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22subPropertyOf%22%3A%7B%22%40id%22%3A%22rdfs%3AsubPropertyOf%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22imports%22%3A%7B%22%40id%22%3A%22owl%3Aimports%22%2C%22%40type%22%3A%22%40id%22%7D%2C%22versionInfo%22%3A%7B%22%40id%22%3A%22owl%3AversionInfo%22%2C%22%40type%22%3A%22%40id%22%7D%7D) to see how this comes out. If there is another templating engine which can generate HTML using the JSON-LD as an input that can generate a human readable version of the vocabulary that would work better with the existing dependencies, then cool. Otherwise, there is the Ruby-based Haml engine used for the JSON-LD vocabulary generation.